walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -77,58 +79,62 @@
           Collections.sort(eventTimes, TimeRange.ORDER_BY_START);
           possibleTimes = findTimeRangeGaps(eventTimes, duration);
           if (!possibleTimes.isEmpty()) {
-            // We've found a combination where scheduling is possible, so those are the desired time slot(s).
+            // We've found a combination where scheduling is possible, so those are the desired time
+            // slot(s).
             return possibleTimes;
           }
           // Re-initialize unavailable time ranges with those of the mandatory attendees.
           eventTimes = new ArrayList<>(attendeesEventTimes);
         }
       }
-      // Calculate time ranges when there are no optional attendees, or none of the optional attendees can fit the schedule.
+      // Calculate time ranges when there are no optional attendees, or none of the optional
+      // attendees can fit the schedule.
       possibleTimes = findTimeRangeGaps(eventTimes, duration);
     }
     return possibleTimes;
   }
 
   /**
-   * Recursive helper function used to find all combinations of size r of the input array arr[].
-   * @param arr[] The input array.
-   * @param chosen[] Temporary array to store current combination.
-   * @param start Starting index in arr[].
-   * @param end Ending index in arr[].
-   * @param index Current index in chosen[].
+   * Recursive helper function used to find all combinations of size r of the elements of a given list.
+   * @param inputList The input list of optional attendees that is immutable.
+   * @param currentCombination Temporary array to store current combination of optional attendees.
+   * @param startIndex Starting index in the input array to consider for the current combination.
+   * @param endIndex Ending index in the input array to consider for the current combination.
+   * @param index Current index to be set in the current combination list.
    * @param r The size of combinations to be found.
    */
-  private void combinationUtil(String arr[], String chosen[], int start, int end, int index, int r) { 
+  private void combinationUtil(
+      ImmutableList<String> inputList, ArrayList<String> currentCombination, int startIndex, int endIndex, int index, int r) {
```

Comment:
You might need to wrap this at 80 chars.

Reply:
I though we said 100 chars for Java but yeah it's over 100